### ISSUE TITLE
Support wireless controller by Retro Bit

### ISSUE THREAD

**Issue by borti4938** [borti4938/Switchless-Mods#3]
The controller is a shame :( However, with the current implementation the uIGR does not work with this controller. I'll try to change it but I cannot promise anything.

Updates regarding the implementation: https://github.com/borti4938/Switchless-Mods/blob/master/SNES/README.RetroBitCtrl

**Comment by borti4938** [comment]
reopen #2

**Comment by borti4938** [comment]
similar issue to #2

**Comment by borti4938** [comment]
Today I have tested a version which seems to solve this problem. I do some further tests and then upload the new version.

**Comment by borti4938** [comment]
These controllers have a long delay from signalling a button bit change by the SNES until the bit is set on the dataline. This is not a problem if the auto joypad read (AJR) is enabled. However, if AJR is not, this may harm the respondence of the uIGR (as the uIGR might not have enough time to store the button bit). One example I found was the Super Gameboy: changing modes was awful. Hence I decided to introduce a new finger breaking button combo to switch on and off the long delays (and hence the support of retro-bit wireless controllers).
This combo will be:
 - L+R+St+B+Y+A+X

The uIGR will confirm with color codes:
- support switched on : green -> yellow -> green
- support switched off: red   -> yellow -> red

I hope this will be an acceptable solution for most people :)
